ui/src/templates/actions/creators.ts

```diff
@@ -11,7 +11,8 @@ export const ADD_TEMPLATE_SUMMARY = 'ADD_TEMPLATE_SUMMARY'
 export const GET_TEMPLATE_SUMMARIES_FOR_ORG = 'GET_TEMPLATE_SUMMARIES_FOR_ORG'
 export const POPULATE_TEMPLATE_SUMMARIES = 'POPULATE_TEMPLATE_SUMMARIES'
 export const REMOVE_TEMPLATE_SUMMARY = 'REMOVE_TEMPLATE_SUMMARY'
-export const SET_ACTIVE_COMMUNITY_TEMPLATE = 'SET_ACTIVE_COMMUNITY_TEMPLATE'
+export const SET_COMMUNITY_TEMPLATE_TO_INSTALL =
+  'SET_COMMUNITY_TEMPLATE_TO_INSTALL'
```

Comment:
"Community Template" feels super long but also necessary for the time being. Will there be a point at which we can go back and rename this stuff "Template"?

Reply:
not until we resolve the two template concepts in the UI !

Reply:
good question. i've been calling these community templates to distinguish them from regular templates while they're both live in the wild. if we end up going with the community template implementation, i think a final step in turning off this feature flag and making them part of the system would be renaming them to just Templates. once the old templates go away, we should start calling these just `templates`, yeah.